pandas/core/generic.py

```diff
@@ -2801,14 +2801,16 @@ def notnull(self):
         """
```

Comment:
pls make sure that you have ``core.filemode=False`` in git

e.g. ``git config global --list`` (then turn it off). you generally don't change file modes (as they are set so that nose works correctly)